### PR TITLE
feat(react-file-type-icons): add portfolio, copilot and update CDN (Sep 2024 filetype icon update])

### PR DESCRIPTION
**Pull request by bigbadcapers**
- [x] updated CDN url to access new filetype icons
- [x] updated mapping file and icontype enum
- [x] `yarn nx ... build` was run to update reported API (new CDN url location)
- [x] tested within @fluentui/react-experiments 
- [x] `yarn change` run
- [x] `yarn nx format` run

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32918&commitId=8eccdd74465f931a53cebe79c4c91e9637a887ad&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/BaseButton-mount_base.html">645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/BaseButton-mount.html">623</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Breadcrumb-mount_base.html">1653</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Breadcrumb-mount.html">1675</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Checkbox-mount_base.html">1679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Checkbox-mount.html">1688</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/CheckboxBase-mount_base.html">1494</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/CheckboxBase-mount.html">1501</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ChoiceGroup-mount_base.html">2946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ChoiceGroup-mount.html">2944</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ComboBox-mount_base.html">685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ComboBox-mount.html">681</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/CommandBar-mount_base.html">6547</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/CommandBar-mount.html">6584</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ContextualMenu-mount_base.html">11635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ContextualMenu-mount.html">11629</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DefaultButton-mount_base.html">798</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DefaultButton-mount.html">797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DetailsRow-mount_base.html">2198</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DetailsRow-mount.html">2257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DetailsRowFast-mount_base.html">2234</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DetailsRowFast-mount.html">2175</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DetailsRowNoStyles-mount_base.html">2038</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DetailsRowNoStyles-mount.html">2030</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Dialog-mount_base.html">2790</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Dialog-mount.html">2665</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DocumentCardTitle-mount_base.html">228</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/DocumentCardTitle-mount.html">225</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Dropdown-mount_base.html">2001</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Dropdown-mount.html">1972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/FocusTrapZone-mount_base.html">1179</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/FocusTrapZone-mount.html">1170</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/FocusZone-mount_base.html">1084</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/FocusZone-mount.html">1050</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedList-mount_base.html">42177</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedList-mount.html">42110</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedList-virtual-rerender_base.html">20027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedList-virtual-rerender.html">20278</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">51407</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedList-virtual-rerender-with-unmount.html">51896</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedListV2-mount_base.html">231</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedListV2-mount.html">237</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedListV2-virtual-rerender_base.html">209</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedListV2-virtual-rerender.html">209</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">234</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">228</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/IconButton-mount_base.html">1098</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/IconButton-mount.html">1137</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Label-mount_base.html">350</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Label-mount.html">345</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Layer-mount_base.html">2717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Layer-mount.html">2797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Link-mount_base.html">398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Link-mount.html">404</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/MenuButton-mount_base.html">963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/MenuButton-mount.html">964</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/MessageBar-mount_base.html">21302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/MessageBar-mount.html">21167</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Nav-mount_base.html">1995</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Nav-mount.html">2064</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/OverflowSet-mount_base.html">795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/OverflowSet-mount.html">798</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Panel-mount_base.html">2080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Panel-mount.html">1844</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Persona-mount_base.html">770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Persona-mount.html">743</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Pivot-mount_base.html">934</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Pivot-mount.html">929</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/PrimaryButton-mount_base.html">935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/PrimaryButton-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Rating-mount_base.html">4752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Rating-mount.html">4733</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SearchBox-mount_base.html">917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SearchBox-mount.html">946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Shimmer-mount_base.html">1861</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Shimmer-mount.html">1891</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Slider-mount_base.html">1347</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Slider-mount.html">1323</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SpinButton-mount_base.html">2959</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SpinButton-mount.html">3009</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Spinner-mount_base.html">401</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Spinner-mount.html">398</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SplitButton-mount_base.html">1841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SplitButton-mount.html">1874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Stack-mount_base.html">415</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Stack-mount.html">444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/StackWithIntrinsicChildren-mount_base.html">888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/StackWithIntrinsicChildren-mount.html">878</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/StackWithTextChildren-mount_base.html">2778</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/StackWithTextChildren-mount.html">2837</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SwatchColorPicker-mount_base.html">6416</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/SwatchColorPicker-mount.html">6442</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/TagPicker-mount_base.html">1456</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/TagPicker-mount.html">1440</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Text-mount_base.html">389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Text-mount.html">394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/TextField-mount_base.html">957</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/TextField-mount.html">948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ThemeProvider-mount_base.html">850</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ThemeProvider-mount.html">846</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ThemeProvider-virtual-rerender_base.html">581</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ThemeProvider-virtual-rerender.html">594</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1299</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1293</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Toggle-mount_base.html">617</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/Toggle-mount.html">631</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/buttonNative-mount_base.html">192</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32918/perf-test/buttonNative-mount.html">192</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json (hunk 1 above)
Comment (bigbadcapers) [reply change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json:1]: false positive, marked as noise in the tool

**Comment by Hotell** [comment]
pipeline is failing -> formatting is off.  run `yarn nx format` to fix it

**Comment by bigbadcapers** [comment]
> 2 things:
> 
> * pipeline is failing -> formatting is off.  run `yarn nx format` to fix it
> * can we update PR title in order to match the changes ? eg `feat(react-file-type-icons): add portfolio, copilot and update CDN`

done! ty @Hotell !

File: change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32918&commitId=9eb7ff110e295ca2baef34c5e5bb9d6fa32b287e&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (bigbadcapers) [reply change/@fluentui-react-file-type-icons-4b02a3cb-433c-4e04-a104-ccd0c48b49b4.json:1]: checked, tagged as noise

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by Jahnp — APPROVED** [review]

**Comment by bigbadcapers** [comment]
hi @spmonahan , @tomi-msft happy friday! could one of you help with the merge for this file icon update (related to Project & Copilot), tyia!

**Review by spmonahan — APPROVED** [review]

**Pull request merged**